docs/source/UserGuide/Tutorials/ShorthandCommands/index.rst

```diff
@@ -0,0 +1,36 @@
+==================
+Shorthand Commands
+==================
+
+These commands are aimed at frequest users of a DIRAC UI, especially if you experience latency issues, as this setup offers a caching mechanism.
+They do require some initial setup.
+
+#. Setting up your default
+
+You only need to do this once.
+
+    $ source diracos/diracosrc
+    $ dirac-proxy-init -g [your DIRAC group]
+    $ dconfig --minimal
+
+This will create a skeleton config file in ``~/.dirac/dcommands.conf``
+Now edit the configuration file usig your details. If you are a member of more than on VO, you might want to setup different profiles for each.
```

Comment:
```suggestion
Now edit the configuration file using your details. If you are a member of more than one VO, you might want to setup different profiles for each.
```

Reply:
Of course I just hit "commit code" and now I have a CI test failure again, as the default commit messages are not DIRAC compliant. Sigh.